src/App.js

```diff
@@ -1,25 +1,76 @@
-import logo from './logo.svg';
-import './App.css';
+
+import "./App.css";
+import contacts from "./contacts.json";
+ import React, { useState } from 'react'; 
 
 function App() {
+  const [contactList, setContactList] = useState(contacts.slice(0, 5))
+  const [randomContact, setRandomContact] = useState(contacts.slice(6))
+
+
+ 
+
+  const newContact = () => { 
+    const newContact = contacts[Math.floor(Math.random()* contacts.length)] 
```

Comment:
If you want to get a random contact that is not listed, you should apply the Math random on randomContact array, not on the original one.